docs/resources/fabric_external_connection_policy.md

```diff
@@ -0,0 +1,176 @@
+
+### Optional ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+  - Default: `uni/tn-infra`
+* `annotation` (annotation) - (string) The annotation of the Fabric External Connection Policy object. This attribute is supported in ACI versions: 3.2(1l) and later.
+  - Default: `orchestrator:terraform`
+* `description` (descr) - (string) The description of the Fabric External Connection Policy object.
+* `name` (name) - (string) The name of the Fabric External Connection Policy object.
+* `name_alias` (nameAlias) - (string) The name alias of the Fabric External Connection Policy object. This attribute is supported in ACI versions: 2.2(1k) and later.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `community` (rt) - (string) A global route target that is used to define the communities for route leaking or redistribution. It can be used in a multi-site deployment to manage routing policies across fabrics.
+* `site_id` (siteId) - (string) A unique identifier for the site associated with the Fabric External Connection Policy object. This attribute is supported in ACI versions: 3.0(1k) and later.
+  - Default: `0`
+* `peering_profile` - (map) A map of BGP EVPN Peering Profile (ACI object [fvPeeringP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvPeeringP/overview)).
+  #### Optional ####
+    
+    * `annotation` (annotation) - (string) The annotation of the BGP EVPN Peering Profile object. This attribute is supported in ACI versions: 3.2(1l) and later.
+      - Default: `orchestrator:terraform`
+    * `description` (descr) - (string) The description of the BGP EVPN Peering Profile object.
+    * `name` (name) - (string) The name of the BGP EVPN Peering Profile object.
+    * `name_alias` (nameAlias) - (string) The name alias of the BGP EVPN Peering Profile object. This attribute is supported in ACI versions: 2.2(1k) and later.
+    * `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+    * `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+    * `password` (password) - (string) The password used for establishing automatic BGP peering sessions. Note that this is not used when the peering type is set to manual mode.
+    * `type` (type) - (string) Specifies the type of BGP EVPN Peering Profile object or component. This helps define how the profile will be applied within the fabric.
```

Comment:
should we remove or component, since we only reference object?

```suggestion
    * `type` (type) - (string) The type of BGP EVPN Peering Profile object.
```